pages/common/buku.md

```diff
@@ -0,0 +1,24 @@
+# buku
+
+> Browser-independent bookmark manager for CLI.
+> More information: <https://github.com/jarun/Buku>.
+
+- Display all bookmarks matching "keyword" and with "privacy" tag:
+
+`buku {{keyword}} --stag {{privacy}}`
+
+- Add bookmark with tags "search engine" and "privacy":
+
+`buku --add {{https://example.com}}} {{search engine}}, {{privacy}}`
```

Comment:
Does `search engine` need quotes around it here?

Reply:
No, the various tags are separated by the comma.

Reply:
Oh, I see! That's unusual.